### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by ellewisnicki**
## 🎯 Objective

Making this change in order to ensure we have breakfast for our three person morning time routine. I am not sure about the cinnamon, BBQ sauce, or milk options.

Acceptance criteria:
1. All steps are included
2. Flavor is considered 
3. Dietary restrictions considered

## 🛠️ Changes made

I'm proposing an entirely new pancake recipe that I pulled from my brain from having a mom that cooks.

## 👥 Who needs to review this Pull Request? 
@eornellas please review, @royberkeley please review

### CODE REVIEW

**Comment by ellewisnicki** [comment]
I think this is a great pancake recipe, but I really want your buy in. Please consider my thoughts in the pull request.

**Comment by royberkeley** [comment]
Hey Elle, I believe people typically don't eat pancakes with BBQ sauce. We should consider just maple syrup instead

File: breakfast/readme.md [hunk 1]
```diff
@@ -1,21 +1,44 @@
...
+## Time to prepare: 30 minutes
 
 ## Ingredients:
-
+1 Egg
+1 cup of milk
+Pancake mix
+Tablespoon of olive oil
+Cinnamon
+BBQ Sauce
```
Comment (eornellas) [inline breakfast/readme.md:13]: Think we should remove BBQ sauce from the list of ingredients

File: breakfast/readme.md [hunk 2]
```diff
@@ -1,21 +1,44 @@
...
+Butter
 
 ## Equipment needed:
-
+Bowl
+Fork
+Whisk
+Spoon
+Pan
+Flipper
```
Comment (eornellas) [inline breakfast/readme.md:22]: Should we change to "spatula"?

File: breakfast/readme.md (hunk 1 above)
Comment (royberkeley) [reply breakfast/readme.md:13]: Agreeeed

Comment (royberkeley) [reply breakfast/readme.md]: Let's start a review

File: breakfast/readme.md (hunk 2 above)
Comment (royberkeley) [reply breakfast/readme.md]: Yes we should

**Comment by royberkeley** [comment]
Ok looks good now

**Pull request merged**